Makefile

```diff
@@ -76,14 +76,10 @@ prepare:
 	mkdir -p tmp
 
 .PHONY: prereqs
-prereqs: ## Test if prerequisites are met, and installing missing dependencies
-	@echo "### Test if prerequisites are met, and installing missing dependencies"
-ifeq (, $(shell which golangci-lint))
+prereqs: ## Install dependencies
+	@echo "### Installing dependencies"
 	GOFLAGS="" go install github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_LINT_VERSION}
-endif
-ifeq (, $(shell which yq))
 	GOFLAGS="" go install github.com/mikefarah/yq/v4@${YQ_VERSION}
-endif
```

Comment:
If you already have yq, it will not update it.
That's an issue since we need at least 4.43.1

See https://redhat-internal.slack.com/archives/C03SHCD5C9M/p1734345188324779?thread_ts=1734110256.626139&cid=C03SHCD5C9M